tests/tests.schema

```diff
@@ -3908,16 +3908,19 @@
       "$role": "interface",
       "oneOf": [
         {
-          "type": "string"
+          "$ref": "#/definitions/CustomAction.dialog"
```

Comment:
>$ref": "#/definitions/CustomAction.dialog" [](start = 11, length = 42)

I don't think you have anything in this PR that should change this file.  I suspect your bf is out of date.